Tools/Scripts/build-api-tests

```diff
@@ -63,15 +63,8 @@ chdirWebKit();
 
 my $result;
 if (isAppleCocoaWebKit()) {
-    chdir "Source/ThirdParty/gtest";
-    buildXCodeProject("xcode/gtest", $clean, XcodeOptions(), @ARGV);
-    chdir "../../../Tools/TestWebKitAPI" or die;
-
-    my @options = XcodeOptions();
-    if ($wtfOnly) {
-        push @options, ("-target", "TestWTF");
-    }
-    $result = buildXCodeProject("TestWebKitAPI", $clean, @options, @ARGV);
+    my $scheme = $wtfOnly ? "TestWTF" : "TestWebKitAPI";
```

Comment:
Does this still build gtest? Looking at the xcscheme, I think that the answer is yes, but want to confirm.

Reply:
Yes!